src/streaming/controllers/ScheduleController.js

```diff
@@ -49,14 +49,14 @@ function ScheduleController(config) {
     const abrController = config.abrController;
     const playbackController = config.playbackController;
     const textController = config.textController;
-    const streamInfo = config.streamInfo;
```

Comment:
Why do we need to move this to the setup function?

Reply:
To enable setting streamInfo to null in reset, and thus to avoid executing event listeners body functions to be executed once reset has been called